packages/common/src/users/HubUsers.ts

```diff
@@ -37,9 +37,13 @@ export async function enrichUserSearchResult(
     links: {
       self: "not-implemented",
       siteRelative: "not-implemented",
-      thumbnail: "not-implemented",
+      thumbnail: null,
     },
   };
+  // Group Memberships need this prop
+  if (getProp(user, "memberType")) {
+    result.memberType = getProp(user, "memberType");
+  }
```

Comment:
```suggestion
  if (user?.memberType) {
    result.memberType = user?.memberType;
  }
```

Reply:
Nice. I had to update the signature so it's `user: IUser & Record<string, any>` to get that to pass typechecking... and add another test b/c istanbul can tell theres another condition vs getProp which is opaque for that.